packages/client/lib/client/index.spec.ts

```diff
@@ -117,14 +117,18 @@ describe('Client', () => {
   });
 
   testUtils.testWithClient('connect, ready and end events', async client => {
+    const p1 = once(client, 'connect');
```

Comment:
I think it's better to commentout this test and add a TODO to check whats going on instead of changing it to "work" without knowning why..

Reply:
reasonable.  even more reasonable would be to have CI mark it as a failure to ignore (if possible), but have it error out for real when we do npm run test manually, so it says in our mind.